exercises/numbers/isEven/isEven.js

```diff
@@ -1,39 +1,23 @@
-/**
- * Returns true if the given number is even false otherwise.
- *
- * An integer is even if the remainder is 0 after dividing
- * by 2. Equivalently, an integer is even if it can be written
- * as 2*k for some integer k.
- *
- * If an integer is not even we call it "odd".
- *
- * Even: -10, -12, -104, 0, 16, 18, 2
- * Odd: -11, -13, -103, -1, 1, 17, 19, 3
- *
- * @param {number} num The input number
- * @returns {boolean} True if num is even and false otherwise
- */
+
+let num = {}
+
 function isEven(num) {
-  // Hint: See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Remainder
-  return _____;
+  if (num % 2 === 0)
+    return true;
+  else
+    return false
 }
 
-if (require.main === module) {
+if (require.main) {
   let evenNumbers = [2, 20, 100, 768];
 
   console.log('Running sanity checks for isEven');
 
-  console.log(isEven(0) === true);
-
-  // These should all be even
   for (let num of evenNumbers) {
-    console.log(isEven(num) === true);
+  console.log(isEven(num));
   }
 
-  // If num is even then num + 1 is odd
   for (let num of evenNumbers) {
-    console.log(isEven(num + 1) === false);
+  console.log(isEven(num + 1));
```

Comment:
The output is now a mixture of `true` and `false`. How can you tell from the terminal output alone whether your program is behaving as expected? Do you remember whether the 4th line of output should be `true` or `false`?

By making the output be all `true` only when all the return values are what we expect, we shape our environment so that we can tell right away when something is off. One `false` line anywhere means something is broken.